lib/api/cln/cln_client.dart

```diff
@@ -259,4 +261,31 @@ class CLNApi extends AppApi {
             jsonResponse as Map<String, dynamic>,
             snackCase: !mode.withCamelCase()));
   }
+
+  @override
+  Future<AppListPeers> listPeers(String id) {
+    dynamic params;
+    switch (mode) {
+      case ClientMode.grpc:
+        params = CLNListPeersRequest(
+            grpcRequest: ListpeersRequest(id: id as List<int>));
```

Comment:
```suggestion
            grpcRequest: ListpeersRequest(id: id as List<int>));
```

List of int? this is odd, maybe you want convert the string to bytes?

Reply:
I don't know about this actually. The `id` should be in string only but the `api` required the use use of `List<int>`

Reply:
Yea but this cast does not work, this is the correct way to this conversion https://stackoverflow.com/a/54844226/10854225